tests/kafkatest/services/zookeeper.py

```diff
@@ -83,3 +88,11 @@ def clean_node(self, node):
 
     def connect_setting(self):
         return ','.join([node.account.hostname + ':2181' for node in self.nodes])
+
+    def set_cmd_opts(self, options):
+        ZookeeperService.KAFKA_OPTS = options
+
+    def zookeeper_migration(self, node, zk_acl):
+        la_migra_cmd = "/opt/%s/bin/zookeeper-security-migration.sh --zookeeper.acl=%s --zookeeper.connect=%s" % (kafka_dir(node), zk_acl, self.connect_setting())
+        node.account.ssh(la_migra_cmd)
+        time.sleep(5)
```

Comment:
I know there is one use of sleep in this file already. In fact it would be best if we could get rid of all calls to `sleep` in this class and instead use `wait_until` to block until some heuristic check on the service state is satisfied as other service classes do